docs/attribute.md

```diff
@@ -0,0 +1,37 @@
+Annotation
+-------------------------
+You can use annotations for checking the feature state in controllers. Just active this in your config:
```

Comment:
Attributes ... and not necessary to activate attributes.

Reply:
It's removed.